newsletter/admin.py

```diff
@@ -228,11 +245,22 @@ class ArticleInline(AdminImageMixin, StackedInline):
         }),
     )
 
+    # Perform any formfield overrides depending on specified settings
+    _formfield_overrides = {}
+
     if newsletter_settings.RICHTEXT_WIDGET:
-        formfield_overrides = {
+        _formfield_overrides = {
             models.TextField: {'widget': newsletter_settings.RICHTEXT_WIDGET},
         }
 
+    # https://easy-thumbnails.readthedocs.io/en/latest/usage/#forms
+    if newsletter_settings.THUMBNAIL == 'easy-thumbnails':
+        _formfield_overrides = {
```

Comment:
This *seems* to override the `_formfield_overrides` when a RichtextWidget is set.

It's also not clear to me why a separate `_formield_overrides` is used instead of directly setting/updating `formfield_overrides`.

Reply:
That is correct. Ultimately this is largely a personal stylistic choice so that we only have one spot where we set the attribute value, but we could achieve the same effect with if/elif/else statements and set the value directly.

Reply:
I would prefer setting them directly as I feel non-functional code does not contribute to clarity.

Reply:
Updated with requested changes.